converters/download

```diff
@@ -3,40 +3,50 @@
 from subprocess import Popen, PIPE
 from argparse import ArgumentParser, FileType
 import os
+import requests
 import sys
 import urllib.parse
-import urllib.request
-import http.client
-
 
 YES, NO, ONLY = "yes", "no", "only"
 
 
-# From SO:
-# https://stackoverflow.com/questions/1875052/using-client-certificates-with-urllib2
-class HTTPSClientAuthHandler(urllib.request.HTTPSHandler):
-    def __init__(self, key, cert):
-        urllib.request.HTTPSHandler.__init__(self)
-        self.key = key
-        self.cert = cert
-
-    def https_open(self, req):
-        # Rather than pass in a reference to a connection class, we pass in
-        # a reference to a function which, for all intents and purposes,
-        # will behave as a constructor
-        return self.do_open(self.getConnection, req)
-
-    def getConnection(self, host, timeout=300):
-        return http.client.HTTPSConnection(host, key_file=self.key, cert_file=self.cert)
-
-
 def download(url, args) -> str:
     if args.auth:
-        opener = urllib.request.build_opener(HTTPSClientAuthHandler(args.key, args.cert))
-        response = opener.open(url)
+        session = get_auth_session(args.cert, args.key)
+        return session.get(url).text
     else:
-        response = urllib.request.urlopen(url)
-    return response.read().decode("utf-8")
+        return requests.get(url).text
+
+
+def get_auth_session(cert=None, key=None):
+    # thanks Brian
+    """
+    Return a requests session ready for an XML query.
+    """
+    if not cert or not key:
+        if "X509_USER_PROXY" in os.environ:
+            cert = key = os.environ["X509_USER_PROXY"]
+        else:
+            euid = os.geteuid()
+            proxypath = "/tmp/x509up_u%d" % euid
+            if euid == 0:
+                cert = '/etc/grid-security/hostcert.pem'
+                key = '/etc/grid-security/hostkey.pem'
+            elif os.path.exists(proxypath):
+                cert = key = proxypath
+            else:
+                cert = os.path.expanduser("~/.globus/usercert.pem")
+                key = os.path.expanduser("~/.globus/userkey.pem")
```

Comment:
Can we add a TODO to respect `X509_USER_CERT` and `X509_USER_KEY` env vars?

Reply:
That's quick enough to fix. Updated.